src/components/Search/index.module.scss

```diff
@@ -0,0 +1,18 @@
+.byNameOrId {
+  all: unset;
+  display: flex;
+  padding: 2px 8px;
+  justify-content: center;
+  align-items: center;
+  margin-right: 16px;
+  color: #666;
+  font-size: 14px;
+  opacity: 0.8;
+  cursor: pointer;
+  user-select: none;
+  border-radius: 2px;
+  background: #f0f0f0;
+
+  // text-wrap: nowrap doesn't work in preview page, try add !important
+  text-wrap: nowrap !important;
```

Comment:
We should avoid using `!important` as much as possible. I recently refactored and removed several instances of code that used `!important`.

First, we need to understand why `text-wrap: nowrap` is not taking effect. Upon inspection, it appears that the reason is that `text-wrap` is being sorted before `all: unset` during compilation. The reason behind this change in compilation sorting behavior is not clear whether it is a feature or a specific behavior of the compilation process. However, I believe we can solve this problem relatively easily with two possible solutions:

1. Avoid using `all: unset`. If we want a cleaner approach, we can use a div to simulate it, for example, by using the `SimpleButton` component.
2. Separate `all: unset` into a separate selector, such as `.buttonReset` or `.byNameOrId`, and set it to `unset`. Then, set `.byNameOrId.byNameOrId` to normal properties. However, if we choose this approach, we need to provide some comments to explain the intention of the code.

Reply:
updated:[b8edbe8](https://github.com/Magickbase/ckb-explorer-frontend/pull/131/commits/b8edbe839cf5a1f6555d08e7bd74efe1ccbba802)